packages/babel-cli/src/babel/file.ts

```diff
@@ -8,17 +8,24 @@ import * as util from "./util";
 import type { CmdOptions } from "./options";
 import * as watcher from "./watcher";
 
+import type {
+  SectionedSourceMap,
+  SectionedSourceMapInput,
+} from "@jridgewell/trace-mapping";
+
+type Section = SectionedSourceMap["sections"][0];
+
 type CompilationOutput = {
   code: string;
-  map: any;
+  map: SectionedSourceMapInput;
 };
 
 export default async function ({
   cliOptions,
   babelOptions,
 }: CmdOptions): Promise<void> {
   function buildResult(fileResults: Array<any>): CompilationOutput {
-    const mapSections = [];
+    const mapSections: Section[] = [];
```

Comment:
Nit: this could just be (and we could get rid of `Section`):
```suggestion
    const mapSections: SectionedSourceMap["sections"] = [];
```